src/main/java/edu/kis/powp/jobs2d/features/TransformationFeature.java

```diff
@@ -22,20 +22,19 @@ public static void setupTransformationPlugin(Application application, DriverMana
         app = application;
         app.addComponentMenu(TransformationFeature.class, "Transformations");
 
-        ToggleTransformationDriverSubscriber subscriber = new ToggleTransformationDriverSubscriber(driverManager, transformationDriver);
+        ToggleDriverSubscriber subscriber = new ToggleDriverSubscriber(driverManager);
```

Comment:
As ToggleDriverSubscriber is now independent (from transformations) it can be moved to DriverFeature. 
Please make the subscriber instance accessible (e.g. singleton) for the whole application, because all features can use it.